File: Java/DataTypesExample.java
//Java datatypes

public class DataTypesExample {
    public static void main(String[] args) {

        int employeeId = 101;
        String employeeName = "Aman";
        double basicSalary = 35000.75;
        double bonus = 5000.50;
        char grade = 'A';
        boolean isPermanent = true;

        double totalSalary = basicSalary + bonus;

        System.out.println("Employee ID      : " + employeeId);
        System.out.println("Employee Name    : " + employeeName);
        System.out.println("Grade            : " + grade);
        System.out.println("Permanent Staff  : " + isPermanent);
        System.out.println("Total Salary     : " + totalSalary);
    }
}

File: Kotlin/DataTypesExample.kt
//Kotlin datatypes

fun main() {

    //val = immutable, var = mutable
    val employeeId:Int = 101
    val employeeName:String = "Aman"
    val basicSalary:Double = 35000.75
    val bonus:Float = 5000.50f  //suffix f for the value else it will be considered Double
    val grade:Char = 'A'
    val isPermanent:Boolean = true

    val totalSalary = basicSalary + bonus

    println("Employee ID      : $employeeId")
    println("Employee Name    : $employeeName")
    println("Grade            : $grade")
    println("Permanent Staff  : $isPermanent")
    println("Total Salary     : $totalSalary")

    //variables can also be declared as (val/var variableName = value). Kotlin automatically detects it.
}